src/libopensc/card-isoApplet.c

```diff
@@ -235,7 +235,6 @@ isoApplet_init(sc_card_t *card)
 		 * driver. */
 		flags = 0;
 		flags |= SC_ALGORITHM_ECDSA_RAW;
-		flags |= SC_ALGORITHM_ECDSA_HASH_SHA1;
 		flags |= SC_ALGORITHM_ONBOARD_KEY_GEN;
```

Comment:
```suggestion
		flags |= SC_ALGORITHM_ECDSA_HASH_NONE;
		flags |= SC_ALGORITHM_ONBOARD_KEY_GEN;
```

With this flag we should not need most of the other stuff to enable the `ECDSA` mechanism.
I did not tested with the AWS device tester though.

Reply:
This works also with the AWS device tester 👍